metro-config-ejected/index.js

```diff
@@ -7,7 +7,13 @@
 
 import "node-libs-react-native/globals";
```

Comment:
TODO: Figure out if this belongs here or in a modified version of `AppEntry.js` from the `expo` package